tests/test_candidates.py

```diff
@@ -43,33 +40,32 @@ def test_candidate(self):
         response = self._response(
             api.url_for(CandidateView, candidate_id=candidate.candidate_id)
         )
-        self.assertResultsEqual(
-            response['pagination'],
-            {'count': 1, 'page': 1, 'pages': 1, 'per_page': 20})
+        assert response['pagination'] == {'count': 1, 'page': 1, 'pages': 1, 'per_page': 20}
         # we are showing the full history rather than one result
-        self.assertEqual(len(response['results']), 1)
+        assert len(response['results']) == 1
 
         result = response['results'][0]
-        self.assertEqual(result['candidate_id'], candidate.candidate_id)
-        self.assertResultsEqual(result['name'], candidate.name)
+        assert result['candidate_id'] == candidate.candidate_id
+        # # most recent record should be first
+        assert result['name'] == candidate.name
         # #address
-        self.assertEqual(result['address_city'], candidate.address_city)
-        self.assertEqual(result['address_state'], candidate.address_state)
-        self.assertEqual(result['address_street_1'], candidate.address_street_1)
-        self.assertEqual(result['address_zip'], candidate.address_zip)
+        assert result['address_city'] == candidate.address_city
```

Comment:
It seems like there's a little inconsistency here in whether dict values are asserted one per line, or against an expected dict. As long as these asserts are being changed, is there a reason to keep them in the former style, rather than convert them to the latter? Personally, I prefer the latter (as in where you've used `assert_dicts_subset`). Do they both result in comparably helpful errors when they fail?